plugins/HHAnalyzer.cc

```diff
@@ -231,7 +202,10 @@ void HHAnalyzer::analyze(const edm::Event& event, const edm::EventSetup&, const
             // Note that ID and isolation criteria are in both electron and muon loops
             if (!dilep.isOS)
                 continue;
-            if (event.isRealData() && !(leptons[ilep1].hlt_DR_matchedObject < m_hltDRCut && leptons[ilep2].hlt_DR_matchedObject < m_hltDRCut))
+            // Throw event if there is no matched dilepton trigger path
+            if (event.isRealData()
```

Comment:
I need a confirmation here, but isn't this condition inverted? If I read this correctly, it throws event when the legs are matched?!